javascript/QUIZ APP/style.css

```diff
@@ -0,0 +1,84 @@
+/* General styling for the form */
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 2%;
+  font-family: Arial, sans-serif;
+}
+
+/* Styling for each question block */
+form > div {
+  width: 60%;
+  background-color: #f9f9f9;
+  border: 1px solid #ddd;
+  border-radius: 5px;
+  padding: 20px;
+  margin-bottom: 20px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+form > div > div#answers {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+form > div > div#answers > button {
+  width: 100%;
+  padding: 10px;
+  margin: 5px 0;
+  border: none;
+  border-radius: 5px;
+  background-color: #eee;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+}
+
+form > div > div#answers > button:hover {
+  background-color: #ddd;
+}
+
+form > div > div#answers > button.btnoptions {
+  font-size: 16px;
+}
+
+form > div > div#answers > button:nth-child(even) {
+  background-color: #f1f1f1;
+}
+
+form > div > div#answers > button:nth-child(odd) {
+  background-color: #fff;
+}
+
+form > div > div#answers > button:disabled {
+  cursor: not-allowed;
+}
+
+form > div > div#answers > button.Shuffle {
+  background-color: #007bff;
+  color: white;
+  border-radius: 20px;
+  margin-top: 10px;
+}
```

Comment:
I would look into [media queries](https://www.w3schools.com/css/css3_mediaqueries.asp) which can help you style depending on the screen size viewing it.

I can imagine when the browser is smaller (more narrow) then the Shuffle buttons look as you wish but when in big screen are "squashed".